lib/modules/switch_board.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+import React from 'react-native';
+const { findNodeHandle } = React;
+const { ARSwitchBoardModule } = React.NativeModules;
+
+function presentNavigationViewController(component, route) {
```

Comment:
The `RCT_EXPORT_METHOD` macro is to export a native module’s method to the JS-side of the bridge. From [the docs](http://facebook.github.io/react-native/docs/native-modules-ios.html#content):

> The name of the method exported to JavaScript is the native method's name up to the first colon. React Native also defines a macro called `RCT_REMAP_METHOD()` to specify the JavaScript method's name. This is useful when multiple native methods are the same up to the first colon and would have conflicting JavaScript names.

To do the integration with Eigen’s ARSwitchBoard, Eigen needs to implement the callbacks that the example app implements in its AppDelegate file: https://github.com/artsy/emission/pull/63/files#diff-d00d369a092249f7c8764b15b71abfd5R64